webpack.config.js

```diff
@@ -104,21 +104,21 @@ const config = {
 			{
 				test: /style\.s?css$/,
 				include: [
-					/blocks/,
+					/editor\/library/,
```

Comment:
Why not `editor/blocks`? 😃

Reply:
Yes, that's what @aduth and @mtias aree suggesting too, updating